src/Features/VisualBasic/Portable/VBFeaturesResources.resx

```diff
@@ -1239,6 +1239,35 @@ Sub(&lt;parameterList&gt;) &lt;statement&gt;</value>
   <data name="Make_0_inheritable" xml:space="preserve">
     <value>Make '{0}' inheritable</value>
   </data>
+  <data name="Apply_file_header_preferences" xml:space="preserve">
+    <value>Apply file header preferences</value>
+  </data>
```

Comment:
this string seems odd to be vb specific.

Reply:
This one should be common if that is possible

Reply:
ok.  please move to common resources location and reference that instead from both C# and VB.

Reply:
📝 In this case, the common resources location is FeaturesResources.resx.

Reply:
Moved